### DISCUSSION TITLE
(mis)Adventuring

### DISCUSSION THREAD

**Discussion by napoleondatamite** [qmd-lab/closeread#150] [General]
1st time Closereading: 1st go someday next week... What are some opening moves/first discoveries/pitfalls to look out for? 
(P.s. 1st time GitHubbin too).

**Comment by jimjam-slam** [comment]
Hi @Micksahippieburning! Do you mean you're looking for some inspiration for using Closeread?

**Reply by napoleondatamite** [reply]
Yes, please. Gonna recce whatever I can before some time off later in the week when my (mis)adventure with Closeread begins…

**Reply by jimjam-slam** [reply]
Fantastic! First stop is probably our [gallery](https://closeread.dev/gallery/), where we have examples and demos. (We also have a [gallery on our preview docs](https://preview.closeread.dev/gallery/) with a few more that we're still cooking up!).

Good luck, and reach out if you need help!

**Comment by napoleondatamite** [comment]
Cheers for sharing, James. No doubt these links will guide me well. Gonna
delve into those links this week ahead... before venturing to Closeread
('the best thing since sliced bread')
